pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -459,11 +469,13 @@ func (nc *NodeController) recordNodeOfflineEvent(node *api.Node) {
 	nc.recorder.Eventf(ref, "offline", "Node %s is now offline", node.Name)
 }
 
-func (nc NodeController) tryUpdateNodeStatus(node *api.Node) (error, time.Duration, api.NodeCondition) {
+// For a given node checks its conditions and tries to update it. Returns grace period to which given node
+// is entitled, state of last observed Ready Condition, and an error if it ocured.
+func (nc NodeController) tryUpdateNodeStatus(node *api.Node) (time.Duration, api.NodeCondition, error) {
```

Comment:
I don't agree - we're trying to update node status, or to be precise node ready condition. The thing is I think tryUpdateNodeReadyCondition is too specific.